docs/release-1.3/README.md

```diff
@@ -0,0 +1,309 @@
+1. Install Orchestrator operator
+   1. Go to OperatorHub in your OpenShift Console.
+   1. Search for and install the Orchestrator Operator.
+1. Create an Orchestrator instance
+   1. Once the Orchestrator Operator is installed, navigate to Installed Operators.
+   1. Select Orchestrator Operator.
+   1. Click on Create Instance to deploy an Orchestrator instance.
+1. Verify resources and wait until they are running
+   1. From console run the following command get the necessary wait commands: \
+      `oc describe orchestrator orchestrator-sample -n openshift-operators | grep -A 10 "Run the following commands to wait until the services are ready:"`\
+
+      The command will return an output similar to the one below, which lists several oc wait commands. This depends on your specific cluster. 
+      ```bash
+        oc wait -n openshift-serverless deploy/knative-openshift --for=condition=Available --timeout=5m
+        oc wait -n knative-eventing knativeeventing/knative-eventing --for=condition=Ready --timeout=5m
+        oc wait -n knative-serving knativeserving/knative-serving --for=condition=Ready --timeout=5m
+        oc wait -n openshift-serverless-logic deploy/logic-operator-rhel8-controller-manager --for=condition=Available --timeout=5m
+        oc wait -n sonataflow-infra sonataflowplatform/sonataflow-platform --for=condition=Succeed --timeout=5m
+        oc wait -n sonataflow-infra deploy/sonataflow-platform-data-index-service --for=condition=Available --timeout=5m
+        oc wait -n sonataflow-infra deploy/sonataflow-platform-jobs-service --for=condition=Available --timeout=5m
+        oc get networkpolicy -n sonataflow-infra
+        ```
+   1. Copy and execute each command from the output in your terminal. These commands ensure that all necessary services and resources in your OpenShift environment are available and running correctly.
+   1. If any service does not become available, verify the logs for that service or consult [troubleshooting steps](https://www.parodos.dev/main/docs/serverless-workflows/troubleshooting/).
+
+### With Helm (deprecated)
+1. Deploy the PostgreSQL reference implementation for persistence support in SonataFlow following these [instructions](https://github.com/parodos-dev/orchestrator-helm-operator/blob/main/docs/postgresql/README.md)
+
+1. Create a namespace for the Orchestrator solution:
+
+   ```console
+   oc new-project orchestrator
+   ```
+
+1. Create a namespace for the Red Hat Developer Hub Operator (RHDH Operator):
+
+   ```console
+   oc new-project rhdh-operator
+   ```
+
+1.  Download the setup script from the github repository and run it to create the RHDH secret and label the GitOps namespaces:
+
+    ```console
+    wget https://raw.githubusercontent.com/parodos-dev/orchestrator-helm-operator/main/hack/setup.sh -O /tmp/setup.sh && chmod u+x /tmp/setup.sh
```

Comment:
Should be `https://raw.githubusercontent.com/parodos-dev/orchestrator-helm-operator/release-1.3/hack/setup.sh`